src/main/java/teammates/ui/controller/Action.java

```diff
@@ -127,17 +127,13 @@ protected AccountAttributes authenticateAndGetActualUser(UserType currentUser) {
         return loggedInUser;
     }
 
+    /**
+     * Retrieves registration key from URL parameter list
```

Comment:
"Parameter list" should not be mentioned. Make it "from the HTTP request", as what the method name suggests. Similarly for the one on the `@return` tag.